srcpkgs/github-cli/template

```diff
@@ -13,7 +13,7 @@ license="MIT"
 homepage="https://cli.github.com"
 changelog="https://github.com/cli/cli/releases"
 distfiles="https://github.com/cli/cli/archive/v${version}.tar.gz"
-checksum=a94ba6a731ad558f7937d0ac46ff8034b56214ec9e24a9ad70296331b1bb12ed
+checksum=1756e59b2db958ff25c264c6c1a1fd86fb224cc84d4043015f759be9b164d988
```

Comment:
Seems like you forgot to update the checksum.

Reply:
Fixed now, but I was pretty sure I changed it a second time as well 🤔